src/Core/Framework/Bundle.php

```diff
@@ -16,7 +17,7 @@
 use Symfony\Component\Routing\RouteCollectionBuilder;
 use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
 
-abstract class Bundle extends SymfonyBundle
+abstract class AbstractPlugin extends SymfonyBundle implements Plugin
```

Comment:
that should stay a bundle because this also used by the Storefront and Administration Bundles

Reply:
okay